<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AudioStar</class>
 <widget class="QWidget" name="AudioStar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>417</width>
    <height>209</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="0,0">
   <item row="2" column="0">
    <widget class="QFrame" name="edge_beatFrame">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="edge_beat_saturation_label">
        <property name="text">
         <string>Saturation</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QTooltipedSlider" name="edge_beat_saturation">
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="edge_beat_hue_label">
        <property name="text">
         <string>Hue</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QTooltipedSlider" name="edge_beat_hue">
        <property name="maximum">
         <number>359</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="edge_beat_sensivity_label">
        <property name="text">
         <string>Beat Sensivity</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QTooltipedSlider" name="edge_beat_sensivity">
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QPushButton" name="audio_settings">
     <property name="text">
      <string>Audio settings</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="edge_beat">
     <property name="text">
      <string>Edge beat</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QTooltipedSlider</class>
   <extends>QSlider</extends>
   <header location="global">QTooltipedSlider.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
